ConsistencyManager/ConsistencyManager.swift

```diff
@@ -140,23 +140,23 @@ open class ConsistencyManager {
      Note that calling this method on a paused listener will not unpause it.
      - parameter listener: The consistency manager listener that is listening to a model
     */
-    open func listenForUpdates(_ listener: ConsistencyManagerListener) {
+    open func addListener(_ listener: ConsistencyManagerListener) {
```

Comment:
This seems most similar to `addObserver`, which is still `addObserver` in Swift 3.

Swift 3 style seems to be either `func add(_ something: Something)` or `func addSomething(_ something: Something)` based on type obviousness, but not `func add(something: Something)`.